viz.yaml

```diff
@@ -115,27 +115,113 @@ fetch:
     scripts: [scripts/fetch/map_data.R , scripts/fetch/utils.R]
     depends: viewbox_limits
 process:
+  -
+    id: precip_grid
+    location: cache/precip_grid.rds
+    reader: rds
+    cell_size: 80000
+    processor: grid_from_mask
+    scripts: [scripts/process/grid_from_mask.R]
+    depends: 
+      clip_poly: "state_map_data"
+  - 
+    id: precip_geoms_dataframe
+    location: cache/precip_geoms_dataframe.rds
+    reader: rds
+    processor: sfc_to_svg
+    attributes:
+      id: '{{ID}}' 
+      class: 'precip-cell'
+    depends: 
+      clip_box: "viewbox_limits"
+      spatial_data: "precip_grid"
+    scripts: [scripts/process/geom_dataframe.R]
   - 
     id: state_geoms_dataframe
     location: cache/state_geoms_dataframe.rds
     reader: rds
-    processor: geom_dataframe
+    processor: sfc_to_svg
     attributes:
-      class: state
-      mouseover: hovertext('{{ID}}', evt)
-      mouseout: hovertext(' ')
+      id: '{{ID}}-geom' 
     depends: 
       clip_box: "viewbox_limits"
       spatial_data: "state_map_data"
     scripts: [scripts/process/geom_dataframe.R]
-  -
-    id: precip_grid
-    location: cache/precip_grid.rds
+  - 
+    id: pr_geoms_dataframe
+    location: cache/pr_geoms_dataframe.rds
     reader: rds
-    cell_size: 80000
-    processor: grid_from_mask
-    scripts: [scripts/process/grid_from_mask.R]
+    processor: sfc_to_svg
+    attributes:
+      class: island
     depends: 
-      clip_poly: "state_map_data"
+      clip_box: "viewbox_limits"
+      spatial_data: "puerto_rico_map_data"
+    scripts: [scripts/process/geom_dataframe.R]
+  - 
+    id: state_style_dataframe
+    location: cache/state_style_dataframe.rds
+    reader: rds
+    processor: dataframe_to_svg
+    attributes:
+      class: state
+      href: '#{{ID}}-geom'
+      id: '{{ID}}'
+    depends: 
+      clip_box: "viewbox_limits"
+      spatial_data: "state_map_data"
+    scripts: [scripts/process/geom_dataframe.R]  
+  - 
+    id: state_mouse_dataframe
+    location: cache/state_mouse_dataframe.rds
+    reader: rds
+    processor: dataframe_to_svg
+    attributes:
+      onmouseover: hovertext('{{ID}}', evt)
+      onmouseout: hovertext(' ')
+      class: mouseover
+      href: '#{{ID}}-geom'
+    depends: 
+      clip_box: "viewbox_limits"
+      spatial_data: "state_map_data"
+    scripts: [scripts/process/geom_dataframe.R] 
+
 visualize:
+  -
+    id: svg_defs
```

Comment:
I'm not really following the structure of these ids. Should this be svg_defs_dataframe? Above you use dataframe as the last element of the id, do you mean to have the format at the end?

Reply:
I don't think so...since I expect that this will either return an xml file or xml as `.rds`. This target is mean to include everything that would be in the `<defs>` part of the svg (see structure on my top comment from when I opened the PR).